account/auth_backends.py

```diff
@@ -6,13 +6,31 @@
 from account.utils import get_user_lookup_kwargs
 
 
-class UsernameAuthenticationBackend(ModelBackend):
+User = get_user_model()
+
+
+class AccountModelBackend(ModelBackend):
+    """
+    This authentication backend ensures that the account is always selected
+    on any query with the user, so we don't issue extra unnecessary queries
+    """
+
+    def get_user(self, user_id):
+        """Get the user and select account at the same time"""
+        user = User._default_manager.filter(pk=user_id).select_related('account').first()
```

Comment:
Will this work with custom user model?

Reply:
Yes, it's using the User model defined in settings, which is what `get_user_model` does.

Reply:
I mean is _default_manager defined on any User models, including custom?

Reply:
It's basically taken from the django code > https://github.com/django/django/blob/ca9872905559026af82000e46cde6f7dedc897b6/django/contrib/auth/backends.py#L155-L160

So should be fine.